scripts/_common.sh

```diff
@@ -4,10 +4,81 @@
 # COMMON VARIABLES
 #=================================================
 
+# dependencies used by the app
+pkg_dependencies="postgresql postgresql-contrib pgloader"
+
 #=================================================
 # PERSONAL HELPERS
 #=================================================
 
+mariadb-to-pg() {
+
+		ynh_print_info --message="Migrating to PostgreSQL database..."
+
+		mysqlpwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
+        
+        # In old instance db_user is `mmuser`
+        mysql_db_user="$db_user"
+        if ynh_mysql_connect_as --user="mmuser" --password="$mysqlpwd" 2> /dev/null <<< ";"; then
+            mysql_db_user="mmuser"
+        fi
+
+        # Initialize PostgreSQL database
+		ynh_psql_test_if_first_run
```

Comment:
This helper is not documented on https://yunohost.org/oc/packaging_apps_helpers, is it new?